internal/plugin/config_plugin_test.go

```diff
@@ -183,16 +188,19 @@ func TestConfig_Update(t *testing.T) {
 	agentConfig := types.GetAgentConfig()
 	instance := protos.GetNginxOssInstance()
 
-	location := fmt.Sprintf("/instance/%s/files/", protos.GetNginxOssInstance().GetInstanceMeta().GetInstanceId())
-	request := model.InstanceConfigUpdateRequest{
-		Instance: instance,
-		Location: location,
+	request := &v1.ManagementPlaneRequest_ConfigApplyRequest{
+		ConfigApplyRequest: &v1.ConfigApplyRequest{
```

Comment:
can you add this payload (v1.ManagementPlaneRequest_ConfigApplyRequest) to the test helpers?